sre/src/sre/plan/compiler.clj

```diff
@@ -1,13 +1,18 @@
 (ns sre.plan.compiler
```

Comment:
I suggest renaming _compiler_ to _planner_.

Reply:
I actually like this name more becuase `sre.plan.planner` sounds a bit repetitive to me. Although I renamed the class `sre.plan.compiler.Compiler` to `sre.plan.compiler.SearchPlanCompiler` and its method `compile` to `calculate`.

Reply:
Another option would be to rename the whole stuff to `sre.planner`, but then the other package names are nouns like `sre.estimation` and `sre.execution` , so that would create another problem. Also I want to keep the nesting of the `dsl` namespace.

Reply:
what do you thin @imbur ?

Reply:
OK, I accept your reasoning for the name.